frontend/app/components/api/api-v3/hal-resources/work-package-resource.service.ts

```diff
@@ -69,27 +69,27 @@ function wpResource(HalResource:typeof op.HalResource, NotificationsService:any,
       this.getForm()
         .catch(deferred.reject)
         .then(form => {
-          var plain_payload = form.payload.$source;
+          var plainPayload = form.payload.$plain();
```

Comment:
I added this as a fix to one of the breaking specs.
I guess it is obsolete, as you have found the actual issues.

I'll remove the commit and push again.